app/views/govuk_publishing_components/components/docs/breadcrumbs.yml

```diff
@@ -6,9 +6,11 @@ body: |
   We recommend that if using the breadcrumbs for navigation purposes, you set `collapse_on_mobile` to `true` to make things more readable for mobile users. However, you can specify `collapse_on_mobile:``false` or remove the flag completely to stop this behaviour.
 shared_accessibility_criteria:
   - link
-accessibility_criteria:
+accessibility_criteria: |
   The breadcrumb links must have a text contrast ratio higher than 4.5:1 against the background colour to meet WCAG AA
   (this especially applies when [using the inverse flag](/component-guide/breadcrumbs/inverse)).
+
+  Always place breadcrumbs at the top of a page, before the `<main>` element. Placing them here means that the "Skip to main content" link allows the user to skip all navigation links, including breadcrumbs. This relates to WCAG 2.2 success criterion [2.4.1 Bypass Blocks (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks).
```

Comment:
Isn't this line too long?

Reply:
Also, there are cases of breadcrumbs being used twice on the page, so maybe "always" is too strong? Maybe something like: "Always place the top-level breadcrumbs at the top of the page, before the \`\<main\>\` element... This does not apply to secondary breadcrumbs which are e.g. below the title."

Reply:
> there are cases of breadcrumbs being used twice on the page

I don't _think_ we know if two sets of breadcrumbs on a page is a pattern we want to encourage, or at least we don't know enough about it to document it here yet. We do have the one known occurrence two sets of breadcrumbs on the HMRC internal manuals page but as Martin as pointed out separately, there might be an accessibility issue with that implementation which we'd like to investigate but that is out of scope here. The Design System doesn't document two sets of breadcrumbs on a page as a design pattern. 

I think the best thing to do here would be to document the standard use of breadcrumbs as Martin has proposed. If we find out later that in fact the use of two sets of breadcrumbs is something we'd like to encourage and we understand the accessibility implications of doing so, then we could come back to this piece of documentation to update it. Reaching out to the accessibility community sounds like a good idea, but maybe doing it as part of that separate card two investigate two sets of breadcrumbs on a page.